iceoryx_examples/iceoptions/README.md

```diff
@@ -72,11 +72,20 @@ would encounter an overflow, the oldest sample is released to create space for t
 subscriberOptions.queueCapacity = 10U;
 ```
 
-`historyRequest` will enable a subscriber to receive the last n samples on subscription e.g. in case it was started later than the publisher. The publisher needs to have its `historyCapacity` enabled, too.
+`historyRequest` will enable a subscriber to receive the last n samples on subscription e.g. in case it was started later than the publisher.
+If the publisher does not have a sufficient `historyCapacity` (larger than `historyRequest`), it will still be connected but we will not be able to
+receive the requested amount of historical data (if it was available). Instead we will receive the largest amount of historical sample
+the publisher has available, i.e. best-effort.
+
+If we want to enforce the contract that the publisher needs to support a `historyCapacity` of at least `historyRequest`, we can do so by setting
+`requirePublisherHistorySupport` to `true`. In this case the subscriber will only connect if the publisher history support is at least as large as is request.
```

Comment:
What if the publisher history support is large enough but does not contain the requested amount of samples?

Reply:
They will connect. And why shouldn't they, it just means this data was never available. If at any time enough samples where available the publisher will have the latest samples available from then on.

It will just refuse connection if it is *never* possible to fulfill the contract, no matter how many samples it has published. This is well-defined I think.